lib/omniauth/strategies/qualtrics.rb

```diff
@@ -19,26 +18,31 @@ class Qualtrics < OmniAuth::Strategies::OAuth2
 
       info do
         {
-          "url" => access_token.client.site
+          'url' => access_token.client.site
         }
       end
 
+      def callback_url
+        full_host + script_name + callback_path
+      end
```

Comment:
We got an error saying it was missing required parameters which is the same error we got with SurveyMonkey so we tried the same remedy and it worked

Reply:
is this overriding a method from super or something?

Reply:
Here's the same change we made for SurveyMonkey https://github.com/user-interviews/omniauth-surveymonkey-oauth2/pull/1

Reply:
twitch integration when